pyproject.toml

```diff
@@ -47,6 +47,8 @@ dependencies = [
     "jsonschema == 4.17.*",
     "pywin32 == 306; sys_platform == 'win32'",
     "QtPy == 2.4.*",
+    "retry >= 0.9.2",
+    "types-retry >= 0.9.2",
```

Comment:
The addition of dependencies affects the [THIRD-PARTY-LICENSES](https://github.com/aws-deadline/deadline-cloud/blob/mainline/THIRD_PARTY_LICENSES) file. Please update that as well.

Reply:
Updated! Chatted offline as suggested